src/test/java/io/odh/test/e2e/upgrade/OlmUpgradeST.java

```diff
@@ -99,14 +99,15 @@ void testUpgradeOlm() throws IOException, InterruptedException {
 
         LOGGER.info("Upgrade to next available version in OLM catalog");
         // Approve upgrade to newer version
-        KubeUtils.waitForInstallPlan(olmInstall.getNamespace(), olmInstall.getCsvName());
-        ip = ResourceManager.getKubeClient().getNonApprovedInstallPlan(olmInstall.getNamespace(), olmInstall.getCsvName());
-        ResourceManager.getKubeClient().approveInstallPlan(olmInstall.getNamespace(), ip.getMetadata().getName());
+        KubernetesUtils.waitForInstallPlan(olmInstall.getNamespace(), olmInstall.getCsvName());
```

Comment:
Having both `KubernetesUtils` and `KubeUtils` seems somewhat confusing.

Reply:
I did get approve for a draft here in the past, even :D Just now I felt like writing down one or two thoughts. Feel free to ignore, ofc 😉